src/binding.cc

```diff
@@ -684,81 +669,80 @@ class Client : public ObjectWrap {
           cur_query.column_count = n_fields;
           for (f = 0; f < n_fields; ++f) {
             field = fields[f];
-            cur_query.column_names[f] = Persistent<String>::New(
-                                            String::New(field.name,
-                                                        field.name_length)
+            NanAssignPersistent(cur_query.column_names[f],
+                                            NanNew<String>(field.name, field.name_length)
                                         );
```

Comment:
Style nit: The above line should be re-indented to match the changes preceding it.